python/ray/_private/gcs_utils.py

```diff
@@ -577,3 +577,36 @@ def use_gcs_for_bootstrap():
     This function is included for the purposes of backwards compatibility.
     """
     return True
+
+
+def cleanup_redis_storage(host: str, port: int, password: str, use_ssl: bool, storage_namespace: str):
+    """This function is used to cleanup the storage. Before we having
+    a good design for storage backend, it can be used to delete the old
+    data. It support redis cluster and non cluster mode.
+
+    Args:
+       host: The host address of the Redis.
+       port: The port of the Redis.
+       password: The password of the Redis.
+       use_ssl: Whether to encrypt the connection.
+       storage_namespace: The namespace of the storage to be deleted.
```

Comment:
can we make these all default to the same environment variables that the GCS does?

Reply:
so called `ray start` and this utility with the same env variables "just works"

Reply:
I don't think it's doable unless we baked everything into OSS envs. I'd say maybe app layer can have a wrapper on top of this (host/port can be retrieved from RAY_REDIS_ADDRESS and storage_namespace can be retrieved from RAY_external_storage_namespace). These should be given by the user.

For password it seems not good (you might not need to use this if it's a closed network).
I'm trying to make the utils not deep coupled with the existing ray api since the current API might be updated in the future (we'll make it backward compatible) and as you can see the current API is really bad, for example, redis address and redis password are passed to ray in different style.

Let me know if it's hard to do it in app layer and we'll see what we can do.

Reply:
Yes the current API is definitely pretty bad lol.

Ok, I think it's fine to not pull them from the env vars.